src/internal/fn.js

```diff
@@ -1,5 +1,10 @@
 import Z from 'sanctuary-type-classes';
 import inspectf from 'inspect-f';
+import * as bc from './bc';
+
+const setImmediate = typeof global.setImmediate === 'function'
+  ? global.setImmediate
+  : /* istanbul ignore next: environment-specific */ bc.setImmediate;
```

Comment:
Is `environment-specific` significant or simply a comment?

Reply:
Simply a comment.